ip/Zmods/ZmodAWGController/constr/generate_timing_xdc.xit

```diff
@@ -42,9 +44,9 @@ puts_ipfile $f_xdc {#Length matching tolerance is 1mm on Eclypse Z7 + 0.1mm on Z
 puts_ipfile $f_xdc {set t_length_matching_err 0.008;}
 puts_ipfile $f_xdc {}
 
-puts_ipfile $f_xdc {set_output_delay -clock [get_clocks ZmodDAC_ClkIO] -clock_fall -min -add_delay [expr $t_hold_falling - $t_length_matching_err] [get_ports {dZmodDAC_Data[*]} -prop_thru_buffers]}
-puts_ipfile $f_xdc {set_output_delay -clock [get_clocks ZmodDAC_ClkIO] -clock_fall -max -add_delay [expr $t_setup_falling + $t_length_matching_err] [get_ports {dZmodDAC_Data[*]} -prop_thru_buffers]}
-puts_ipfile $f_xdc {set_output_delay -clock [get_clocks ZmodDAC_ClkIO] -min -add_delay [expr $t_hold_rising - $t_length_matching_err] [get_ports {dZmodDAC_Data[*]} -prop_thru_buffers]}
-puts_ipfile $f_xdc {set_output_delay -clock [get_clocks ZmodDAC_ClkIO] -max -add_delay [expr $t_setup_rising + $t_length_matching_err] [get_ports {dZmodDAC_Data[*]} -prop_thru_buffers]}
+puts_ipfile $f_xdc {set_output_delay -clock [get_clocks nameZmodDAC_ClkIO] -clock_fall -min -add_delay [expr $t_hold_falling - $t_length_matching_err] [get_ports {dZmodDAC_Data[*]} -prop_thru_buffers]}
```

Comment:
47-50 are missing corresponding string maps to replace the nameZmodDAC_ClkIO template.